src/components/organisms/phone-number-us.tsx

```diff
@@ -0,0 +1,149 @@
+import React, {FC, useState, useRef} from 'react';
```

Comment:
A copy of the previous phone-number component that has the country code removed. Also a new asset key was created for this.